daemon/environment.cpp

```diff
@@ -589,6 +591,10 @@ pid_t start_install_environment(const std::string &basename, const std::string &
         log_perror("dup2 failed");
     }
 
+    if (-1 == setpriority(PRIO_PROCESS, 0, extract_priority)){
```

Comment:
serve.cpp handle_connection() we use 
'''
int niceval = nice(nice_level);
    if (niceval == -1) {
        log_warning() << "failed to set nice value: " << strerror(errno)
                      << endl;
    }
'''

Seems like we need to be consistent